src/modules/commentsCounter.js

```diff
@@ -0,0 +1,6 @@
+const commentCounter = () => {
+  const comments = document.querySelectorAll('.putComments');
+  return comments.length;
```

Comment:
Good job implementing the function, but think about a scenario where there is no element i.e. comment in the DOM. In line with this, I think you should take care of that scenario and return 0, in order to further improve the feedback given to user concerning what's going on the UI.